readme.txt

```diff
@@ -3,7 +3,7 @@ Contributors: globalpayments
 Tags: woocommerce, woo, unified, commerce, platform, global, payments, ucp, heartland, payment, systems, tsys, genius, 3DS, gateway, token, tokenize, save cards
 Requires at least: 5.4
 Tested up to: 5.8
```

Comment:
After testing and prior to release we'll want to update this to reflect the current WC version

Reply:
updated the version details